features/time.yml

```diff
@@ -0,0 +1,12 @@
+# TODO: remove after https://github.com/mdn/browser-compat-data/pull/23638
+status:
```

Comment:
No order is enforced, but I'll move this to where it is by convention in most files, closer to the end.